elements/lisk-dpos/src/delegates_list.ts

```diff
@@ -63,7 +103,9 @@ class DelegatesList {
 		return shuffleDelegateListForRound(round, delegatePublicKeys);
 	}
 
-	async getDelegatePublicKeysSortedByVoteWeight(tx) {
+	public async getDelegatePublicKeysSortedByVoteWeight(
+		tx?: StorageTransaction,
```

Comment:
if this `tx` refers to a database transaction it would be better to find a better name to avoid the conflict with the term `transaction` on our domain

Reply:
oh well maybe this was already like this so it's ok if too much effort to rename now.